### PR TITLE
Docs: add description for timepicker fields

### PR DESCRIPTION
**Pull request by someden**

### CODE REVIEW

**Comment by someden** [comment]
@imatwawana Is there anything else I need to do in this PR?

**Comment by imatwawana** [comment]
Hi @someden, I was tagged for review because this PR affects the documentation, but it requires review from an engineer, so I requested review from the Dashboards team. I'll ping the team.

File: docs/sources/dashboards/build-dashboards/view-dashboard-json-model/index.md
Comment (axelavargas) [inline docs/sources/dashboards/build-dashboards/view-dashboard-json-model/index.md:169]: In this other [PR](https://github.com/grafana/grafana/pull/79703/files#diff-b468897532069e2eea891a159f8136398cb7a9869a17d8f8ae376fdaf2800022R203) we just added a better description of this property in our dashboard schema, this can be used here :) 


```suggestion
| **nowDelay**          | override the now time by entering a time delay. Use this option to accommodate known delays in data aggregation to avoid null values.               |
```

**Review by axelavargas — COMMENTED** [review]
Thank you for contributing to Grafana @someden 🏅.  I left a minor suggestion :)

**Review by axelavargas — APPROVED** [review]
🥳 awesome! Thank you for making the changes. I will merge the PR when the CI passes.

**Pull request merged**
